src/server/migrations/1_create_initial_tables.sql

```diff
@@ -20,8 +20,8 @@ CREATE TABLE node (
 );
 
 CREATE TABLE edge (
-  source_id INTEGER REFERENCES node(id),
-  target_id INTEGER REFERENCES node(id),
+  source_id INTEGER REFERENCES node(id) ON DELETE CASCADE,
```

Comment:
The migration system change is now merged, so SQL changes have to be done as proper migrations.

Reply:
It seems to be working for me after merging main. Do I have to do something else?